### DISCUSSION TITLE
2022-21: 远程工作的光谱

### DISCUSSION THREAD

**Discussion by giscus** [Xuanwo/blog#352] [Announcements]
# 2022-21: 远程工作的光谱

An infrastructure engineer, focused on distributed storage system

https://xuanwo.io/reports/2022-21/

**Comment by gexiao** [comment]
公司予以的补贴需要达到税前工资的 61% 以上才能覆盖个人通过其他渠道缴纳同等五险一金的开销。
这个计算的比例不对，只需要计算公司应缴的部分，个人的部分本来就是要从工资里扣掉的。

我认为成立分公司是最稳妥简单的方案，找个当地代理一把包掉，流程上麻烦点其他都还好，一年几千块成本比薪资支出低多了，就看公司有没有诚意。

**Reply by Xuanwo** [reply]
> 这个计算的比例不对，只需要计算公司应缴的部分，个人的部分本来就是要从工资里扣掉的。

确实，我改一下原文～

Fixed in https://github.com/Xuanwo/blog/commit/df73b421c2222bdadbdfba0ace2a98e5a1347609
